emukit/examples/models/random_forest.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, X_init: np.ndarray, Y_init: np.ndarray, num_trees: int = 30,
         self._Y = Y_init
 
         self.rf = RandomForestRegressor(n_estimators=num_trees, bootstrap=do_bootstrapping, random_state=seed)
-        self.rf.fit(X_init, Y_init)
+        self.rf.fit(X_init, Y_init.ravel())
```

Comment:
My normal grumble about being careful with dimensions, so use reshape instead of ravel of flatten, goes here. But since this is an example, I am not going to be super hard on it

Reply:
Changed to `[:, 0]`